website/static/img/needle.svg

```diff
@@ -1,75 +1,19 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
-<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Or this

Reply:
<img width="638" alt="Screen Shot 2020-12-14 at 1 53 44 PM" src="https://user-images.githubusercontent.com/727224/102122670-d263f900-3e13-11eb-843a-a63ef2e0d8bf.png">

Reply:
striken.